app/views/pages/index.html.erb

```diff
@@ -1,4 +1,23 @@
 <div class="container text-center">
   <h1>Pages#index</h1>
   <p>Find me in app/views/pages/index.html.erb</p>
+  <% if current_user %>
+    <h2>Logged-in User:</h2>
+    <hr>
+    <%= current_user.username %><br>
+    <%= current_user.email %>
+    <h2>Welcome, stranger!</h2><br>
+    <%= link_to 'Log Out', session_path, method: :delete %>
+    <%= link_to 'Edit Profile', edit_user_path(current_user) %>
+  <% else %>
+    <h2>Welcome, stranger!</h2>
+      <div class="row">
+        <div class="col col-lg-1 offset-lg-5">
```

Comment:
Consistency is key. At the end of the day quotations doesnt really matter so long as you are consistent.